Week2/ramazanInce.js

```diff
@@ -0,0 +1,158 @@
+console.log(`
+Question 1`);
```

Comment:
Hi Ramazan, was there a reason you preferred to use back-ticks here and split the string to two lines?

Reply:
Yes, when you run the code I want to put an empty line between the questions.